packages/block-library/src/group/block.json

```diff
@@ -35,7 +35,13 @@
 			"color": true,
 			"radius": true,
 			"style": true,
-			"width": true
+			"width": true,
+			"__experimentalDefaultControls": {
+				"color": true,
+				"radius": true,
```

Comment:
Making a note to self:

What is the minimum we need here? Is radius a candidate for an optional control?

Related comment: https://github.com/WordPress/gutenberg/pull/33743#issuecomment-972743640